src/sql/src/session/vars.rs

```diff
@@ -959,6 +959,15 @@ const KEEP_N_SOURCE_STATUS_HISTORY_ENTRIES: ServerVar<usize> = ServerVar {
     internal: true
 };
 
+/// Whether compute rendering should use Materialize's custom linear join implementation rather
+/// than the one from Differential Dataflow.
+const STORAGE_CLIENT_FINALIZE_SHARDS: ServerVar<bool> = ServerVar {
+    name: UncasedStr::new("storage_client_finalize_shards"),
```

Comment:
I can't find the link right now, but I think the style suggestion is to phrase as `enable_` and default false / override true